web_01.html

```diff
@@ -1,68 +1,66 @@
-                                    <li><a href="#">SHOP</a></li>
-                                    <li><a href="#">BLOG</a></li>
-                                    <li><a href="#">FEATURES +</a>
-                                        <ul>
-                                            <li><a href="#">TEMPLATE FEATURES</a></li>
-                                            <li><a href="#">TESTIMONIALS</a></li>
-                                            <li><a href="#">OUT STAFF</a></li>
-                                            <li><a href="#">404 NOT FOUND</a></li>
-                                        </ul>
-                                    </li>
-                                    <li><a href="#">CONTACT +</a></li>
-                                    <li><a href="#"><i class="fa-solid fa-bag-shopping"></i> $180</a>
-                                        <ul>
-                                            <li><a href="#"><img src="/images/bag.jpg" width="40px">2 x $20 = $40</a></li>
-                                            <li><a href="#"><img src="/images/woman.jpg" width="40px">2 x $20 = $40</a></li>
-                                        </ul>
-                                    </li>
-                                </ul>
-                            </nav>
+                        <nav>
+                            <ul>
+                                <li><a href="#">HOME</a>
+                                    <ul>
+                                        <li><a href="#" class="active">CLASSIC STYLE</a></li>
+                                        <li><a href="#">PORTFOLIO</a></li>
+                                        <li><a href="#">SHOWCASE</a></li>
+                                        <li><a href="#">VIDEO PLAYER</a></li>
+                                    </ul>
+                                </li>
+                                <li><a href="#">SHOP</a></li>
+                                <li><a href="#">BLOG</a></li>
+                                <li><a href="#" >FEATURES</a>
+                                    <ul>
+                                        <li><a href="#">TEMPLATE FEATURES</a></li>
+                                        <li><a href="#">TESTIMONIALS</a></li>
+                                        <li><a href="#">OUT STAFF</a></li>
+                                        <li><a href="#">404 NOT FOUND</a></li>
+                                    </ul>
+                                </li>
+                                <li><a href="#">CONTACT</a></li>
+                                <li><a href="#"><i class="fa-solid fa-bag-shopping"></i> $180</a>
+                                    <ul>
+                                        <li><a href="#"><img src="/images/bag.jpg" width="40px"></img>2 x $20 = $40</a></li>
+                                        <li><a href="#"><img src="/images/woman.jpg" width="40px"></img>2 x $20 = $40</a></li>
```

Comment:
`img` tag doesn't have children and it is a self-closing tag similar to `<hr />`, `<link />` and etc hence it should look like this: `<img />`